MobSF/views/home.py

```diff
@@ -298,6 +298,9 @@ def delete_scan(request, api=False):
                         if isDirExists(item_path) and item.startswith(md5_hash + "-"):
                             shutil.rmtree(item_path)
                     data = {'deleted': 'yes'}
+                else:
+                    if api:
+                        return {'deleted': 'not_found'}

```

Comment:
Is this required?
Why not provide it here: https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/master/MobSF/views/home.py#L276

You might need to change the JS Logic in template as well

Reply:
In theory, it should return a 404 status code. When I call this API, I don't know if this is not deleted because the error still doesn't exist, because I may have deleted it through the page, so I need to distinguish some states.

Reply:
Hmf. Let me test the behaviour once.

Reply:
@magaofei I am seeing the JSON response message for already deleted items

```
curl -X POST --url http://localhost:8000/api/v1/delete_scan --data "hash=3a552566097a8de588b8184b059b0158" -H "Authorization:***"
{"deleted": "no"}
```

Reply:
What is that you see as an issue here?

Reply:
When deleting a project that does not exist, it should return a state that does not exist, which means that there is no such project, not the deletion error.